src/Bicep.LangServer/Handlers/BicepSignatureHelpHandler.cs

```diff
@@ -186,7 +186,7 @@ private static SignatureHelp CreateSignatureHelp(ImmutableArray<FunctionArgument
             return null;
         }
 
-        private static SignatureInformation CreateSignature(FunctionOverload overload, ImmutableArray<FunctionArgumentSyntax> arguments, bool includeReturnType)
+        private static SignatureInformation CreateSignature(SemanticModel model, FunctionOverload overload, ImmutableArray<FunctionArgumentSyntax> arguments, bool includeReturnType)
```

Comment:
SemanticModel model seems not be used anymore, looks like leftover after some refactoring